# HG changeset patch
# User Rik <rik@octave.org>
# Date 1378267884 25200
#      Tue Sep 03 21:11:24 2013 -0700
# Node ID e0b9d43395e2760f59ad561183f38ab457e8abd0
# Parent  1102c81e9d60fe50318ffd51fb9aaf06c09fdd45
Fix alphabetization in contributors.in (bug #39933)

* doc/interpreter/contributors.in: Fix alphabetizations.  Add Peter Rosin as
contributor (previously known as Peter Ekberg).

diff --git a/doc/interpreter/contributors.in b/doc/interpreter/contributors.in
--- a/doc/interpreter/contributors.in
+++ b/doc/interpreter/contributors.in
@@ -37,19 +37,19 @@ Clinton Chee
 Albert Chin-A-Young
 Carsten Clark
 Catalin Codreanu
 J. D. Cole
 Martin Costabel
 Michael Creel
 Jeff Cunningham
 Martin Dalecki
+Jacob Dawid
 Jorge Barros de Abreu
 Carlo de Falco
-Jacob Dawid
 Thomas D. Dean
 Philippe Defert
 Bill Denney
 Fabian Deutsch
 Christos Dimitrakakis
 Pantxo Diribarne
 Vivek Dogra
 John Donoghue
@@ -156,31 +156,31 @@ Timo Lindfors
 Benjamin Lindner
 Ross Lippert
 David Livings
 Sebastien Loisel
 Erik de Castro Lopo
 Massimo Lorenzin
 Emil Lucretiu
 Hoxide Ma
+Colin Macdonald
 James Macnicol
 Jens-Uwe Mager
-Colin Macdonald
 Rob Mahurin
+Alexander Mamonov
 Ricardo Marranita
 Orestes Mas
 Axel Mathéi
 Makoto Matsumoto
 Tatsuro Matsuoka
+Christoph Mayer
 Laurent Mazet
 G. D. McBain
-Alexander Mamonov
-Christoph Mayer
+Ronald van der Meer
 Júlio Hoffimann Mendes
-Ronald van der Meer
 Thorsten Meyer
 Petr Mikulik
 Mike Miller
 Stefan Monnier
 Antoine Moreau
 Kai P. Mueller
 Hannes Müller
 Victor Munoz
@@ -224,16 +224,17 @@ Balint Reczey
 Joshua Redstone
 Lukas Reichlin
 Michael Reifenberger
 Anthony Richardson
 Jason Riedy
 E. Joshua Rigler
 Petter Risholm
 Matthew W. Roberts
+Peter Rosin
 Andrew Ross
 Fabio Rossi
 Mark van Rossum
 Joe Rothweiler
 Kevin Ruland
 Kristian Rumberg
 Ryan Rusaw
 Olli Saarela
@@ -268,35 +269,35 @@ Russell Standish
 Brett Stewart
 Doug Stewart
 Jonathan Stickel
 Judd Storrs
 Thomas Stuart
 Ivan Sutoris
 John Swensen
 Daisuke Takago
+Ariel Tankus
 Falk Tannhäuser
-Ariel Tankus
+Duncan Temple Lang
 Matthew Tenny
+Kris Thielemans
 Georg Thimm
-Duncan Temple Lang
-Kris Thielemans
 Olaf Till
 Christophe Tournery
 Thomas Treichl
 Karsten Trulsen
 Frederick Umminger
 Utkarsh Upadhyay
-Daniel Wagenaar
 Stefan van der Walt
 Peter Van Wieren
 James R. Van Zandt
 Risto Vanhanen
 Gregory Vanuxem
 Ivana Varekova
+Daniel Wagenaar
 Thomas Walter
 Andreas Weber
 Olaf Weber
 Thomas Weber
 Rik Wehbring
 Bob Weigel
 Andreas Weingessel
 Martin Weiser
